CONTRIBUTING.md

```diff
@@ -104,3 +104,25 @@ Once a change has been posted as a GitHub pull request, a synctos project contri
 Special care should be taken to ensure that each submission is captured as a GitHub issue, thoroughly documented in `README.md` and in `CHANGELOG.md`'s "Unreleased" section, comprehensively covered by test cases, includes examples in the sample document definitions directory, does not introduce breaking changes to public APIs, does not introduce new package dependencies and does not make use of advanced JavaScript/ECMAScript language features that are not supported by the [version](https://github.com/robertkrimen/otto/tree/5282a5a45ba989692b3ae22f730fa6b9dd67662f) of the otto JavaScript engine/interpreter that is used by Sync Gateway.
 
 If/when a change is deemed satisfactory, it is the responsibility of the reviewer to merge the pull request and delete its feature branch, where possible.
+
+# Publishing
+
+Follow these steps when it is time to publish a new release:
+
+1. Create a GitHub issue/ticket for the release (e.g. https://github.com/Kashoo/synctos/issues/272) with the [task](https://github.com/Kashoo/synctos/issues?q=is%3Aissue+label%3Atask) label and assigned to your own GitHub user
+2. Create a release branch (e.g. `v2.2.1-release`)
+3. Create a GitHub release candidate tag (e.g. `v2.2.1-rc.1`) from the HEAD of the release branch; include the version's changelog content in the description and mark it as a "pre-release"
+4. Validate the release candidate with [kashoo-document-definitions](https://github.com/Kashoo/kashoo-document-definitions), for example, by changing the package's "synctos" dependency version to target the release candidate tag (e.g. "git@github.com:Kashoo/synctos.git#v2.2.1-rc.1") and then running `npm install && npm test`. Confirm that a generated sync function also works with live instances of both Sync Gateway 1.x and 2.x.
+5. Create a new branch (e.g. `issue-272-release-2.2.1`) based off of the _release_ branch, rather than the master branch:
+    1. Modify the "Unreleased" section of `CHANGELOG.md` to display the new version number and date stamp
```

Comment:
You might also mention updating the between version history links at the bottom of `CHANGELOG.md` as I know it was easily overlooked when I went through this process myself.

Reply:
Will do.